rfcs/text/0000-extend-file-pe.md

```diff
@@ -0,0 +1,142 @@
+# 0000: Extend the PE field set
+
+- Stage: **1 (proposal)**
+- Date: **TBD**
+
+The Portable Executable (PE) sub-field, of the `file` top-level fieldset, can be updated to include more file attributes to aid in file analysis. This additional document metadata can be used for malware research, as well as coding and other application development efforts.
+
+## Fields
+
+This RFC is to create 25 additional sub-fields within the `file.pe` fieldset.
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| file.pe.authentihash | keyword | Authentihash of the PE file. |
```

Comment:
Should these all be `file.pe` or just `pe` since `pe.*` is not a top-level field but is reused in other contexts, besides just file (`process` and `dll`)

Reply:
Correct, these should be added under the `pe.*` field and then will be nested under `file`, `process`, and `dll`.

Reply:
Fix't